pkg/utils/openstack/OWNERS

```diff
@@ -0,0 +1,7 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
```

Comment:
Please no utils package. This package has a very high chance of becoming collections of various different types of packages and causes long term confusion. Please try and use something that is more specific that doesn't have `utils` as directory in path.